### PR TITLE
NerdFest-Skycoin-Technical-Details.de.md

### PR DESCRIPTION
**Pull request by C3POv7**

### CODE REVIEW

**Comment by C3POv7** [comment]
Dependent on #794

**Comment by gz-c** [comment]
The build failed because the opening section between `+++` is formatted incorrectly

**Comment by C3POv7** [comment]
> The build failed because the opening section between `+++` is formatted incorrectly

I can not detect any difference between the original one and mine.

File: content/infographics/NerdFest-Skycoin-Technical-Details.de.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+length = 282624
++++
+  
+## Skycoin führt die Blockchain-Innovation an. Glauben Sie uns nicht? Überzeugen Sie sich selbst!
+
+* Verbesserungen bei der Netzwerksicherheit
+* Doppelte Coinbase-Outputs wurden eliminiert
+* Sicherheit vor 51-Prozent-Angriffen
+* Unterstützung für das GMaxwell CoinJoin-Protokoll zum Schutz der Privatsphäre
+* Unterschrift-Fälschbarkeit beseitigt
```
Comment (kietob) [inline content/infographics/NerdFest-Skycoin-Technical-Details.de.md]: Fälschbarkeit der Signatur beseitigt

File: content/infographics/NerdFest-Skycoin-Technical-Details.de.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* Beseitigung von Minern
+* Blockheader-Fehlerkorrektur
+* Anpassbare Blockchain-Größe ermöglicht die Skalierung von Netzwerktransaktionen
+* TX / UTXO-Transaktionsstruktur, die Welten besser ist, als die Multigraphs von Bitcoin
```
Comment (kietob) [inline content/infographics/NerdFest-Skycoin-Technical-Details.de.md]: TX/UTXO-

**Comment by C3POv7** [comment]
Ready to merge

**Pull request closed without merging**